### PR TITLE
Improving file structure + added ombd-service

### PR DESCRIPTION
**Pull request by mverdaguer**
Example when searching for ender:

`[ { Title: 'Ender: The Eero Ettala Documentary',
    Year: '2015',
    imdbID: 'tt5089754',
    Type: 'movie',
    Poster: 'N/A' },
  { Title: 'Norm Ender: Tekir',
    Year: '2007',
    imdbID: 'tt8350518',
    Type: 'movie',
    Poster: 'https://m.media-amazon.com/images/M/MV5BZThmYzZkMmItNmFlMy00OTUyLTkyZGItZjQ5OTRkZDI5MWNlXkEyXkFqcGdeQXVyNDg4MjkzNDk@._V1_SX300.jpg' },
  { Title: 'Norm Ender: Içinde Patlar',
    Year: '2010',
    imdbID: 'tt8350492',
    Type: 'movie',
    Poster: 'https://m.media-amazon.com/images/M/MV5BMGNjNWQ5OGYtNGNmYy00NjBiLWI4YjktODJlYTFiZWVkMTFiXkEyXkFqcGdeQXVyNDg4MjkzNDk@._V1_SX300.jpg' },
  { Title: 'Norm Ender: Kinim',
    Year: '2002',
    imdbID: 'tt8350574',
    Type: 'movie',
    Poster: 'https://m.media-amazon.com/images/M/MV5BM2VmMWMyZjQtYTA5OS00NTBiLTg0MzctMDUzNzZkNjA4ODNmXkEyXkFqcGdeQXVyNDg4MjkzNDk@._V1_SX300.jpg' },
  { Title: 'Det ender med bryllup',
    Year: '1943',
    imdbID: 'tt0125074',
    Type: 'movie',
    Poster: 'https://images-na.ssl-images-amazon.com/images/M/MV5BN2YyYjdiZDgtY2VmMy00N2Y4LTkzOTAtN2FlM2ZkZmNmNDNjXkEyXkFqcGdeQXVyMjA0MzYwMDY@._V1_SX300.jpg' },
  { Title: 'i-Report Year Ender Special',
    Year: '2007',
    imdbID: 'tt1160344',
    Type: 'movie',
    Poster: 'N/A' },
  { Title: 'Ender',
    Year: '2013',
    imdbID: 'tt2543532',
    Type: 'movie',
    Poster: 'N/A' },
  { Title: 'Ender-Colossus',
    Year: '2012',
    imdbID: 'tt2637364',
    Type: 'movie',
    Poster: 'N/A' },
  { Title: 'The Ender Dragon Quest',
    Year: '2014',
    imdbID: 'tt4276866',
    Type: 'movie',
    Poster: 'N/A' },
  { Title: 'Det ender med drab',
    Year: '2015',
    imdbID: 'tt5019510',
    Type: 'series',
    Poster: 'N/A' },
  { Title: 'Ender',
    Year: '2013',
    imdbID: 'tt5358174',
    Type: 'movie',
    Poster: 'N/A' } ]`

### CODE REVIEW

File: dist/example.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+"use strict";
```
Comment (MarcPuigdemont) [inline dist/example.js]: This file is ignored by .gitignore, how did it end up here?

File: src/services/ombd-service.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import 'es6-promise';
```
Comment (MarcPuigdemont) [inline src/services/ombd-service.js:1]: Thank god there is a checkbox to hide all comments, right arround ......................... ^ here

Hound is not yer configured, but something is off with the coding style. Can you format the document with prettifier here?

Comment (marcmortensen) [reply src/services/ombd-service.js:1]: +1

File: src/services/ombd-service.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+import 'isomorphic-fetch';
+
+export default class OmbdService {
+  constructor(key) {
+    this.key = key;
+  }
+
+  async search(title, initPage = 1, numPages = 1) {
+    // prettier-ignore
+    const url = `https://www.omdbapi.com?apikey=${this.key}&s=${title}`
```
Comment (MarcPuigdemont) [inline src/services/ombd-service.js]: woof! Missing semicolon
This is because prettier is ignored for this line that it's not added automatically

**Review by MarcPuigdemont — APPROVED** [review]

**Review by laurasanguesa — APPROVED** [review]

**Pull request merged**